src/test/java/net/logstash/logback/encoder/LogstashEncoderTest.java

```diff
@@ -70,12 +66,12 @@
 public class LogstashEncoderTest {
     
     private static Logger LOG = LoggerFactory.getLogger(LogstashEncoderTest.class);
-    
+
```

Comment:
<nitpick> Would you be willing to revert all these whitespace changes?

Reply:
This one is annoying me aswell, I didnt had this change until I commited, something wrong in my Settings I guess, I will look into it.